jwst/refpix/irs2_subtract_reference.py

```diff
@@ -177,6 +178,7 @@ def correct_model(input_model, irs2_model, scipix_n_default=16, refpix_r_default
 
     # Convert corrected data back to sky orientation
     output_model = input_model.copy()
+    input_model.close()
```

Comment:
Why not just continue to use `input_model`, rather than copy it, then close it?

Reply:
thank you for catching this!